etc/genome/spec/lsf_resource_dv2_manta.yaml

```diff
@@ -1,3 +1,3 @@
 ---
-default_value: "-R 'select[tmp>=10000 && mem>=8000] rusage[tmp=10000, mem=8000] span[hosts=1]' -M 8000000 -n 12"
+default_value: "-R 'select[tmp>=10000 && mem>=24000] rusage[tmp=10000, mem=24000] span[hosts=1]' -M 24000000 -n 12"
```

Comment:
Assuming 2GB per job slot.  See the comment about RAM usage below:
https://github.com/Illumina/manta/issues/38